locale/en.po

```diff
@@ -3251,192 +3304,204 @@ msgstr "Responsive"
 msgid "FOCUSED"
 msgstr "Focused"
 
-#: ../src/microbe_stage/editor/CellEditorComponent.GUI.cs:333
-#: ../src/microbe_stage/editor/CellEditorComponent.GUI.cs:338
+#: ../src/microbe_stage/editor/CellEditorComponent.GUI.cs:165
+msgid "DIGESTION_SPEED_VALUE"
+msgstr "{0} n/s"
```

Comment:
What does the '*n*' mean here?

Reply:
n generally means amount, so this can be read: x amount of compounds per second.

Reply:
Yeah I guess it could work.